lib/loader.js

```diff
@@ -35,6 +36,12 @@ class VASTLoader extends EventEmitter {
     })
   }
 
+  fetch (...args) {
+    return this._root._fetch != null
```

Comment:
I would merge this method into line 27, so that:

- The initialization only happens once per instance.
- There's no public method. (This could of course also be solved by prefixing with an underscore.)